config/locales/en.yml

```diff
@@ -69,3 +69,24 @@ en:
     big:
       uk: From 1 January 2021, if you’re bringing goods into Northern Ireland from outside the UK and the EU, you will pay the UK duty rate <a href="https://www.gov.uk/guidance/check-if-you-can-declare-goods-you-bring-into-northern-ireland-not-at-risk-of-moving-to-the-eu-from-1-january-2021" class="govuk-link">if your goods are not ‘at risk’ of onward movement to the EU</a>. If they are at risk of onward movement to the EU, use the %{link}.
       xi: From 1 January 2021, if you’re bringing goods into Northern Ireland from outside the UK and the EU, you will pay the UK duty rate <a href="https://www.gov.uk/guidance/check-if-you-can-declare-goods-you-bring-into-northern-ireland-not-at-risk-of-moving-to-the-eu-from-1-january-2021" class="govuk-link">if your goods are not ‘at risk’ of onward movement to the EU</a>. If they are not at risk of onward movement to the EU, use the %{link}.
+    steel_safeguards: >- # Temporary warning regarding the steel safeguards
```

Comment:
you can say here `steel_safeguards_html`

Reply:
I've changed it to `steel_safeguards_html`.

I've put it here because
* the same pattern is already used in the Duty Calculator footnotes, and
* it's supposed to go away at some point soon, so it's good to have it in a very specific and clearly flagged location

Would you have rather put it into the helper instead?

Reply:
Leave it here for now.